.github/workflows/test.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       ALL_WARNINGS: "/tmp/all-warnings"
 
     strategy:
+      fail-fast: false
```

Comment:
Adding this to see if the failure is limited to 3.1.

I must admit I don't understand what the failure is about, but I'll try to dig.

Reply:
Thanks, I want that setting there!